modules/remove.js

```diff
@@ -0,0 +1,9 @@
+export default class Remove {
+  remove = (id) => {
+    let allBooks = JSON.parse(localStorage.getItem('bookList'));
+    allBooks = allBooks.filter((book) => book.id.toString() !== id.toString());
+    localStorage.setItem('bookList', JSON.stringify(allBooks));
+    // eslint-disable-next-line
```

Comment:
- [ ] Please avoid disabling linter rules, the linters help tremendously to keep a consistent code structure and enforce some best practices over your code! Kindly consider removing this comment and fixing the error. 🤓 🔧